### PR TITLE
ENH add current repodata hack on non-x86_64 for 2.17 in prep for 2.28

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

I am not bumping the build number here since I simply want to make the current repodata packages that don't exist. Then I'll submit a repodata patch to fix up all of the old sysroots to use them.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
@isuruf this one is ready to go!

**Comment by beckermr** [comment]
bump here @isuruf any comments or shall we merge?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -84,25 +84,34 @@ outputs:
...
       run_constrained:
         - sysroot_{{ cross_target_platform }} =={{ version }}
     test:
       commands:
         test -f $PREFIX/{{ target_machine }}-{{ ctng_vendor }}-linux-gnu/sysroot/usr/include/linux/version.h
 
-{% if cross_target_platform == "linux-64" %}
   - name: _sysroot_{{ cross_target_platform }}_curr_repodata_hack
+{% if cross_target_platform == "linux-64" %}
+    # on linux-64 2.12 has the highest version and 2.17/2.28 are lower
```
Comment (isuruf) [inline recipe/meta.yaml:103]: This would be
- 4 -> 2.12
- 3 -> 2.17
- 2 -> 2.28
?

Comment (beckermr) [reply recipe/meta.yaml:103]: We only need the one we want to be the default to be on v4 and the other ones, older or newer, on v3. So for 2.28 we will have v3 as well.

**Review by isuruf — APPROVED** [review]

**Pull request merged**